addon/components/select-light.hbs

```diff
@@ -15,7 +15,11 @@
         <option
           value={{get optionValue this.valueKey}}
           selected={{is-equal (get optionValue this.valueKey) @value}}>
-          {{get optionValue this.displayKey}}
+          {{#if @valueKey}}
```

Comment:
```suggestion
          {{#if (has-block "option")}}
```
Close! I think this could follow the following pattern rather than relying on the seemingly unrelated (but passing the test) `@valueKey`
https://github.com/emberjs/ember.js/pull/19318/files#diff-08e773e2dfe82a277c4837362632f2e6d5d2d18137feaa80456e7d0537ff37d8R185